<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>464</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>317</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">QPushButton { background-color:rgb(58, 71, 81)}</string>
     </property>
     <property name="title">
      <string>Controller Tools</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Scale selected controllers</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="ctrl_scale_1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>0.25</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ctrl_scale_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>0.5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ctrl_scale_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>1.25</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ctrl_scale_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>1.5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Extras</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="ctrl_mirror">
          <property name="text">
           <string>Mirror Ctrl Shape</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ctrl_replace">
          <property name="toolTip">
           <string>Select controller to be replaced, and then controller to replace</string>
          </property>
          <property name="text">
           <string>Replace Controlers</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">QPushButton { background-color:rgb(75, 61, 79)}</string>
     </property>
     <property name="title">
      <string>IK FK Tools</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Please select the joints you want to use.</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Make sure they follow this namig structure:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>{}_skin_jnt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="create_FK">
        <property name="toolTip">
         <string>Creates just a simple FK setup, with controllers</string>
        </property>
        <property name="text">
         <string>Create FK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="create_IK">
        <property name="toolTip">
         <string>Creates just a simple IK setup with pole vector</string>
        </property>
        <property name="text">
         <string>Create IK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="create_IKFK">
        <property name="toolTip">
         <string>Creates BOTH an IK and FK setup, then constraints it with a switch</string>
        </property>
        <property name="text">
         <string>Create IK FK with Switch</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
